packages/cli/create-strapi-app/src/utils/usage.ts

```diff
@@ -57,8 +57,10 @@ function trackEvent(event: string, payload: Record<string, unknown>) {
     return;
   }
 
+  const analyticsUrl = process.env.STRAPI_ANALYTICS_URL || 'https://analytics.strapi.io';
```

Comment:
Do you think it would be possible to have a centralized constant for the string instead of hard-coding it everywhere?

As far as I know we don't currently have anywhere to put constants that are used in multiple packages, and I wouldn't want to add it now since it will take some discussion. So I'm asking because I'm not sure, but I'm curious if you have ideas 😆